src/main/java/controller/EventController.java

```diff
@@ -33,6 +34,24 @@
     /**
      * checks whether the time period is available to schedule a new event
      * and add the event if it is available
+     * @param eventName name of event
+     * @param startDateTime start time including date
+     * @param duration duration of event
+     * @return whether the creation succeeded
+     */
+    public boolean createEvent(String eventName, LocalDateTime startDateTime, Duration duration) {
+        LocalTime endTime = startDateTime.plus(duration.toSeconds(), ChronoUnit.SECONDS).toLocalTime();
```

Comment:
May exceed 24h

Reply:
added note, we might need to use exceptions to achieve this